recipe-server/normandy/recipes/management/commands/update_recipe_signatures.py

```diff
@@ -41,7 +41,7 @@ def handle(self, *args, force=False, **options):
         if count == 0:
             self.stdout.write('No disabled recipes to unsign')
         else:
-            self.stdout.write('Unsigning {} disabled recipes:'.format(count))
+            self.stdout.write(f'Unsigning {count} disabled recipes:')
```

Comment:
"unsigning"?
I know it's not strictly related to this patch, but could you explain what this is meant to provide?